lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/YarnClusterSubmissionFromCS.java

```diff
@@ -181,6 +200,13 @@ String getFileSystemUrl() {
   }
 
   /**
+   * @return The environment map.
+   */
+  Map<String, String> getEnvMap() {
+    return envMap;
```

Comment:
This returns a reference to a mutable collection. Is this safe? Maybe return a copy?

Reply:
Also, consider renaming to `getEnvironmentVariablesMap` or such.

Reply:
Done